babel.config.js

```diff
@@ -0,0 +1,28 @@
+const BROWSER_TARGETS = {
+  browsers: ['last 5 versions', 'FireFox >= 44', 'Safari >= 7', 'Explorer 11', 'last 4 Edge versions'],
```

Comment:
at the beginning of  August 17, 2021, Microsoft will no longer support IE 11..should we wait?

Reply:
Please, does it work? shouldn't it be better to add a browser slug and version?

Reply:
It does work, Babel does it out of the box including few polyfills, like `fetch`. Remove that support won't change much for us to be honest, thus positive impact is almost 0. I'd prefer keep it as long material-ui does it https://material-ui.com/getting-started/supported-platforms/